src/main/java/oss/fosslight/api/controller/v1/ApiOssController.java

```diff
@@ -110,23 +112,47 @@ public CommonResult getOssInfoByDownloadLocation(
     public CommonResult getLicenseInfo(
     		@RequestHeader String _token,
     		@ApiParam(value = "License Name", required = true) @RequestParam(required = true) String licenseName){
-		
+
 		// 사용자 인증
 		userService.checkApiUserAuth(_token);
 		Map<String, Object> resultMap = new HashMap<String, Object>();
-		
+
 		try {
 			List<Map<String, Object>> content = apiOssService.getLicenseInfo(licenseName);
-		
-			
+
+
 			if (content.size() > 0) {
 				resultMap.put("content", content);
 			}
-			
+
 			return responseService.getSingleResult(resultMap);
 		} catch (Exception e) {
 			return responseService.getFailResult(CoConstDef.CD_OPEN_API_UNKNOWN_ERROR_MESSAGE
 					, CoCodeManager.getCodeString(CoConstDef.CD_OPEN_API_MESSAGE, CoConstDef.CD_OPEN_API_UNKNOWN_ERROR_MESSAGE));
 		}
     }
+
+	@ApiOperation(value = "Register New Oss", notes = "신규 Oss 등록")
```

Comment:
```suggestion
	@ApiOperation(value = "Register New OSS", notes = "신규 OSS 등록")
```

OSS = Open Source Software

Reply:
thanks!